### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by davidwalter2**
- Fix running with --excludeProcGroups Fake (previously crashed because fake syst uncertainty was always added and passSystToFakes was False even if --excludeProcGroups Fake was given)
- Fix --pseudoDataFile for wmass (previously no hist selection was performed on pseuodata from extra file)
- Fix --pseudoData for root output (previously pseudodata was overridden with asimov)

### CODE REVIEW

**Comment by cippy** [comment]
This looks fine to me, it should finally skip the fakes when either --filterProcGroups is passed and does not include them, or --excludeProcGroups explicitly includes the fakes

**Comment by davidwalter2** [comment]
Somehow the CI shows up as still running but everything has been successful, so I cancel the CI now. 
The PRValidation:
https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/PR446/2024_05_13/
Has as expected no change to the reference run:
https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/ReferenceRuns/2024_05_10_53426de/
From my side this is ready to be merged

**Comment by cippy** [comment]
I am fine with merging. Unless someone else has comments you can proceed

File: scripts/combine/setupCombine.py [hunk 1]
```diff
@@ -326,6 +326,11 @@ def setup(args, inputFile, fitvar, xnorm=False):
         if args.pseudoDataFile:
             # FIXME: should make sure to apply the same customizations as for the nominal datagroups so far
             pseudodataGroups = Datagroups(args.pseudoDataFile, excludeGroups=excludeGroup, filterGroups=filterGroup)
+            pseudodataGroups.set_histselectors(
```
Comment (bendavid) [inline scripts/combine/setupCombine.py:329]: should there also be ```pseudodataGroups.fakerate_axes=args.fakerateAxes``` somewhere here?

**Pull request merged**